atlasdb-config/src/main/java/com/palantir/atlasdb/http/AtlasDbHttpClients.java

```diff
@@ -83,11 +87,23 @@ private AtlasDbHttpClients() {
             ServerListConfig serverListConfig,
             Class<T> type,
             AuxiliaryRemotingParameters parameters) {
-        Supplier<T> clientFactory = () -> VersionSelectingClients.instrumentWithClientVersionTag(
+        Supplier<T> clientFactory = () -> instrument(
                 metricsManager.getTaggedRegistry(),
                 ConjureJavaRuntimeTargetFactory.DEFAULT.createProxyWithQuickFailoverForTesting(
                         serverListConfig, type, parameters),
                 type);
         return SelfRefreshingProxy.create(clientFactory, type);
     }
+
+    private static <T> T instrument(
```

Comment:
iirc, instrumentation has always been separate from creating clients. i.e. `ServiceCreator` and in timelock (before and after my changes). Unless they're all changed, we're going to double report, and likely we'll have wrong metrics on our dashboards.

Reply:
good catch!

Reply:
Discussed offline: This is messy, because just removing will mean that we lose some metrics - not true that all `ServiceCreator` created things are instrumented. Will do as a follow up to this PR.